_posts/2021-04-09-three-ways-we-supported-the-past-signalen-community-meetup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Three ways we supported the last Signalen community meetup"
+date: 2021-04-09
+author: Alba Roza
+type: blogpost
+excerpt: Organizing a virtual event with the Signalen community
+categories:
+  - Signalen
+---
+
+# Three ways we supported the last Signalen community meetup
+
+On 22 March, one of the codebases we work with, the Dutch [Signalen](https://signalen.org/), organized its first public community meetup. The aim of this virtual get together was to get to know new municipalities curious about the solution for nuisance reporting as well as vendors interested in its implementation.
+
+As part of its community, the Foundation for Public Code helped by advising on mostly three aspects.
+
+## Open Source tools
+
+To this day, Dutch civil servants still work primarily with the tools of a large American corporation - however things are starting to change. The Signalen community meetup was a day to celebrate the success of a public codebase, so why wouldn’t we encourage the use of open technologies to hold the event?
+
+### Eventyay
+
+Following the example of FOSSAsia we decided to use Eventyay for the registration of participants. [Eventyay](https://eventyay.com/) is an open source solution for event organizers that features several options to customize your event, including a scheduler, call for papers, sponsors, etc. 
+ 
+For the meetup, the Signalen community offered newcomers a prepared demo environment to test Signalen beforehand, so it was very important to ask a few questions during the registration process. 
+
+### Jitsi
+
+For a while now the Signalen community has used Jitsi for their regular calls and steering groups and it works just fine, and so it was the chosen tool for the several breakout rooms in the event. 
+
+The machine (hosted in the Foundation for Public Code’s server) did its job well: there were moments with more than 50 attendees in a room, all with good connections.
```

Comment:
I think we should be careful with this statement as I'm sure Jacco has another experience with this as during the end of the meeting we had troubles (but may be these were self inflicted)

Reply:
I see what you mean but I believe that the problem was not the machine but a code/design error.

However, I'm going to rewrite it for a better understanding.